libs/ui/src/components/MediaItem/type/ModelMedia.vue

```diff
@@ -15,7 +15,14 @@
 </template>
 
 <script lang="ts" setup>
-import '@google/model-viewer'
+useHead({
+  script: [
+    {
+      src: 'https://ajax.googleapis.com/ajax/libs/model-viewer/3.3.0/model-viewer.min.js',
+    },
+  ],
+})
+
```

Comment:
why we remove `@google/model-viewer` from package.json and import the script directly? It would be more difficult to manage the version of dependence

Reply:
`import "@google/model-viewer"` throw error in libs/ui (check #7701) I didn't find a way to fix it with import so I changed it with script import and issue is no longer here

Reply:
let me know if you know a workaround (maybe transpile?)

Reply:
This line will affect our UI. The 3D model on this page doesn't work on my end. https://deploy-preview-7702--koda-canary.netlify.app/ksm/gallery/19645919-26A39DB07731155A62-DFW-SOFAGLB_1-00000061

For now, let's mock it for the histoire side. You can create an empty file and add it under alias. We can figure it out later for a proper fix

```js
// histoire.config.ts
alias: {
  '@': resolve(__dirname, '../../'),
  '~': resolve(__dirname, '../../'),
  '@google/model-viewer': './__mocks__/model-viewer.js', // empty file
},
```

I think it's better to mark `model-viewer` as a custom element also on our nuxt.config

```js
// nuxt.config.js
vue: {
  ..., // existing config
  compilerOptions: {
    isCustomElement: (tag) => ['model-viewer'].includes(tag),
  },
},
```

and upgrade `@google/model-viewer` to the latest version

Reply:
> I think it's better to mark `model-viewer` as a custom element also on our nuxt.config

Why that? we don't use directly `model-viewer` in nuxt, its extended from ModelMedia

Reply:
TIL new `@google/model-viewer` version need `three js` as peer dep

Reply:
> > I think it's better to mark `model-viewer` as a custom element also on our nuxt.config
> 
> Why that? we don't use directly `model-viewer` in nuxt, its extended from ModelMedia


to fix this warning in development mode

![image](https://github.com/kodadot/nft-gallery/assets/734428/bf7c54bf-acb9-4f64-814f-5004801a80bc)

iirc Jarsen already fixed this, not sure why it shows up again

Reply:
I try to do:
```
plugins: [
  vue({
    template: {
      compilerOptions: {
        isCustomElement: (tag) => ['model-viewer'].includes(tag),
      },
    },
  }),
],
```
but it still throw warning on my side

Reply:
The config was for `nuxt.config.ts`, I still don't know yet how to fix that on the histoire side

Reply:
`model-viewer` doesn't load any item in histoire preview